src/main/kotlin/org/ionproject/integration/model/internal/timetable/Course.kt

```diff
@@ -0,0 +1,11 @@
+package org.ionproject.integration.model.internal.timetable
+
+data class Course(
+    var acronym: String = "",
+    var type: String = "",
+    var room: String = "",
+    var begin_time: String = "",
+    var end_time: String = "",
+    var duration: String = "",
+    var weekday: String = ""
```

Comment:
these should be vals without a default initial value

Reply:
They can't because they aren't filled all at the same time